packages/esm-patient-medications-app/src/add-drug-order/add-drug-order.workspace.tsx

```diff
@@ -54,7 +53,6 @@ export default function AddDrugOrderWorkspace({ order: initialOrder, closeWorksp
 
   const saveDrugOrder = useCallback(
     (finalizedOrder: DrugOrderBasketItem) => {
-      finalizedOrder.careSetting = config.careSettingUuid;
```

Comment:
We do need the care setting here. However, for now, it can be non-configurable, hard-coded to the default value.